src/python/WMCore/MicroService/Unified/MSRuleCleaner.py

```diff
@@ -0,0 +1,72 @@
+"""
+File       : MSRuleCleaner.py
+
+Description: MSRuleCleaner.py class provides the logic used to clean the Rucio
```

Comment:
I'd provide a more complete description, such as:
```
This MicroService is meant to remove Rucio rules that are no longer needed in the Workload Management system, such as:
 * block-level rules created by WMAgent, against the origin RSE where data is getting produced
 * container and block-level rules created by MSTransferor, for input data that is no longer in the system
In addition to that, this MicroService is now also responsible for the workflow archival, which is the final status that
workflows remain.
```

What do you think?

Reply:
I think your doc-string is perfect. It goes directly into the header. Thanks!